{
  "id": 5600846,
  "key": "MM-T2697",
  "name": "LDAP User Filter error",
  "createdOn": "2020-06-01T17:46:17Z",
  "objective": null,
  "precondition": null,
  "estimatedTime": null,
  "labels": ["se-prod", "ldap"],
  "component": null,
  "priority": { "id": 401951, "name": "Low" },
  "status": { "id": 890281, "name": "Active" },
  "folder": {
    "id": 404269,
    "name": "AD LDAP",
    "parentId": 2954447,
    "fullNames": ["Suite", "AD LDAP"],
    "fullPath": "suite/ad-ldap"
  },
  "customFields": {
    "Playwright": null,
    "Authors": "",
    "Priority P1 to P4": "P4 - Low-Impact (Edge or unsuitable to repeat?)",
    "Update notes": "",
    "Rainforest": ["N/A"],
    "Team Ownership": ["Suite Users"],
    "Manual Test Environments": [],
    "Detox": null,
    "Cypress": "To Do",
    "MMCTL": null,
    "Last Updated": "",
    "Tags": [],
    "Location": "AD LDAP"
  },
  "steps": [
    {
      "inline": {
        "description": "LDAP User Filter error<br>————————————————————————————<ol><li>AD/LDAP Settings ➜ User Filter</li><li>Enter:<br>(&amp;(objectCategory=Person)(!(UserAccountControl:<br><br></li><li>Save</li></ol>",
        "testData": null,
        "expectedResult": "<ul><li>Error message shows above save button, \"Invalid AD/LDAP filter\"</li><li>Set back to:<br>(&amp;(objectCategory=Person)(!(UserAccountControl:1.2.840.113556.1.4.803:=2)))<br>or just switch to a different page and back again and verify it returned to previous value</li></ul>",
        "customFields": {
          "Fix Versions": [],
          "Related ticket(s)": "",
          "Important notes about this step": ""
        }
      },
      "testCase": null
    }
  ],
  "caseHashed": "22651ec21536996e62825225893a0c5cc951a609f8c757f463c34689112ebab05e52c4d6932f97d03a33491570fb52b1",
  "stepsHashed": "ba70722868f0931b5ccb57ae950d82228adcdee78d11ce41edf7545c41524cb92e6f81e869a40d05388ec97eeab740d1"
}
